### PR TITLE
Convert legacy stub

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: client/components/application-summary.js [hunk 1]
```diff
@@ -29,7 +29,8 @@ const mapStateToProps = ({
     user,
     showConditions,
     basename,
-    drafting
+    drafting,
+    project: actualProject
```
Comment (lennym) [inline client/components/application-summary.js]: :-|

Comment (joefitter) [reply client/components/application-summary.js]: 🍆

File: client/components/application-summary.js [hunk 2]
```diff
@@ -49,7 +50,8 @@ const mapStateToProps = ({
     values: project,
     sections: schema(),
     basename,
-    drafting
+    drafting,
+    actualProject
```
Comment (lennym) [inline client/components/application-summary.js]:
```suggestion
    project: actualProject
```

File: client/components/application-summary.js (hunk 1 above)
Comment (joefitter) [reply client/components/application-summary.js]: actualProject_final

File: client/components/application-summary.js [hunk 3]
```diff
@@ -136,6 +138,12 @@ class ApplicationSummary extends React.Component {
     if (this.props.drafting) {
       return window.alert('Submitting to ASRU through this tool is not currently supported.')
     }
+
+    if (this.props.actualProject.isLegacyStub) {
```
Comment (lennym) [inline client/components/application-summary.js]:
```suggestion
    if (this.props.project.isLegacyStub) {
```

**Pull request merged**